config/clients/java/template/src/main/api/client/model/ClientWriteStatus.java.mustache

```diff
@@ -2,21 +2,6 @@
 package {{clientPackage}}.model;
 
 public enum ClientWriteStatus {
-    SUCCESS("success"),
-    FAILURE("failure");
-
-    private final String value;
-
-    ClientWriteStatus(String value) {
-        this.value = value;
-    }
-
-    public String getValue() {
-        return value;
-    }
-
-    @Override
-    public String toString() {
-        return value;
-    }
+    SUCCESS,
+    FAILURE
```

Comment:
This enum doesn't seem to be consistent with other model enums in the SDK, specifically in lacking `@JsonCreator` and `@JsonValue` methods. 

If this enum is a model stemming from the actual API (e..g, like [ConsistencyPreference](https://github.com/openfga/java-sdk/blob/7de889944cef75ac015cf97738bad1c42c873f5c/src/main/java/dev/openfga/sdk/api/model/ConsistencyPreference.java#L21)) it should be consistent with those enums for jackson.

If it's not related to the API model itself I think it should probably be moved out of the model package (see [CredentialsMethod](https://github.com/openfga/java-sdk/blob/7de889944cef75ac015cf97738bad1c42c873f5c/src/main/java/dev/openfga/sdk/api/configuration/CredentialsMethod.java#L18))